src/site/template/aurelia/layouts/widget/menubar/default.php

```diff
@@ -12,26 +12,29 @@
 
 defined('_JEXEC') or die();
 
-use Kunena\Forum\Libraries\Icons\KunenaIcons;
+use Joomla\CMS\Language\Text;
 
 ?>
 
```

Comment:
Hi!
I do not know what I'm doing wrong ;)
If I change that, I'll lose my registration. Everything is moved to left side.
![Screenshot_157](https://user-images.githubusercontent.com/115731598/196290915-c6461c37-4ed1-4e4e-87f0-e08548210215.jpg)
![Screenshot_158](https://user-images.githubusercontent.com/115731598/196290920-f6345136-e8d3-4866-963e-d0dcce43d598.jpg)